sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,851 @@
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 8279, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+
+-- Naga Explorer 1907 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2090, 1907, -13669.1, 616.477, 7.57179, 3.58152, 300, 300, 5, 100, 1, 10),
+(2093, 1907, -13701, 633.343, 7.3798, 3.06592, 300, 300, 5, 100, 1, 10),
+(2554, 1907, -13705.4, 574.627, 18.7638, 4.46864, 300, 300, 5, 100, 1, 10),
+(180, 1907, -13783.5, 645.277, 14.8763, 5.95696, 300, 300, 5, 100, 1, 10);
+UPDATE `creature` SET `position_x` = -13780.404297, `position_y` = 498.528076, `position_z` = 69.540359, `wander_distance` = 5, `movement_type` = 1 WHERE `guid` = 1966;
+
+-- Tethis 730 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2588, 730, -12978.6, 411.841, 21.7456, 3.43503, 420, 600, 15, 100, 1, 10),
+(565, 730, -13017.2, 193.101, 22.7814, 1.45268, 420, 600, 15, 100, 1, 10);
+UPDATE `creature` SET `wander_distance` = 20, `movement_type` = 1 WHERE `id` = 730;
+DELETE FROM `pool_creature_template` WHERE `id` = 730;
+INSERT INTO `pool_creature_template` (`id`, `pool_entry`, `chance`, `description`, `patch_max`) VALUES
+(730, 125, 0, 'Tethis (730)', 10);
+DELETE FROM `pool_template` WHERE `entry` = 1266;
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(125, 1, 'Tethis (730)');
+
+-- Southsea Cannoneer Should Not Drop Lvl 20 Items
+UPDATE `creature_loot_template` SET `item` = 30041 WHERE `entry` = 3382 AND `item` = 30040 AND `groupid` = 0;
+
+-- Farmer Ray Should Not Drop Lvl 30+ Items
+DELETE FROM `creature_loot_template` WHERE  `item`= 4421 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 4424 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 6149 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 1645 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 3928 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 4419 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 4422 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `item`= 4599 AND `entry`= 232;
+DELETE FROM `creature_loot_template` WHERE  `entry` = 232 AND `item` = 30048;

```

Comment:
Why reverse the order of the 2 arguments here? It doesn't actually matter, but looks weird when you did it the other way for all the above.